Rdmp.Core/Databases/CatalogueDatabase/runAfterCreateDatabase/CreateCatalogue.sql

```diff
@@ -1,4 +1,5 @@
-﻿--Version:1.0.0.0
```

Comment:
Adding a leading blank line? Is version always supposed to be 1.0.0.0 not incremented on changes?